api/tests/unit/features/test_unit_features_views.py

```diff
@@ -2322,6 +2322,39 @@ def test_cannot_update_environment_of_a_feature_state(
     )
 
 
+def test_update_feature_state_without_history_of_fsv(
```

Comment:
Where was the error coming from previously? In the audit log generation? And this test is sufficient I guess because `TASK_RUN_METHOD` is `SYNCHRONOUS`?

Reply:
> Where was the error coming from previously

The error was coming from ` get_update_log_message`

> And this test is sufficient I guess because TASK_RUN_METHOD is SYNCHRONOUS?

Yep, that is correct